src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -355,6 +363,13 @@ private void HandleRegularCommand(StartPageListItem item)
                     dvm.ShowNewFunctionDialogCommand.Execute(null);
                     break;
 
+                case ButtonNames.ShowGalleryUI:
```

Comment:
For all the new things you added in this PR, if they have the term `UI`, can you make them `Ui` instead? For example, `ButtonNames.ShowGalleryUi` here. If you think `UI` can be dropped from the name, feel free too. For example, `ButtonNames.ShowGallery` here.

And many more below...